test.js

```diff
@@ -93,7 +93,7 @@ describe('cpFile()', function () {
 
 	it('should not create dest on unreadable src', function (cb) {
 		cpFile('node_modules', 'tmp', function (err) {
-			assert(err.code === 'EISDIR');
+			assert(err.cause().code === 'EISDIR');
```

Comment:
I agree with you. This is also covered by the author:
> Preserve all of the properties of the original error. … except for name, message, and stack …

Sadly, `verror` does not provide this.